pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/patch/PatchFactory.java

```diff
@@ -24,13 +28,13 @@ public static BasePatch makePatch(Patch patch, NotificationUtil notificationUtil
                 patcher = new TissuePatch(patch);
             } else if (isSmIdCreation(patch)) {
                 patcher = new SMIDPatch(patch);
-            } else if (isParentParticipandDataId(patch)) {
+            } else if (isParentParticipantDataId(patch)) {
                 patcher = new ParticipantDataPatch(patch);
             } else if (isParticipantIdForRecord(patch)) {
                 patcher = new ParticipantRecordPatch(patch);
             }
         } else if (isParentParticipantId(patch) && !isMedicalRecordAbstractionFieldId(patch)
-                    && StringUtils.isNotBlank(patch.getDdpParticipantId())) {
+                && StringUtils.isNotBlank(patch.getDdpParticipantId())) {
```

Comment:
Consider making a `hasDDPParticipantId() method on patch instead of having clients do things like `StringUtils.isNotBlank(patch.getDdpParticipantId())`